src/component/Nav.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
+import { Link } from 'react-router-dom';
+
+const Nav = () => (
+  <header>
+    <nav className='nav_bar'>
+      <div className="logo_head">
+        <img src="/images/rocket.jpeg" alt="logo" className='logo' />
+        <p>Space Travelers' Hub</p>
+      </div>
+      <div className="navigation">
```

Comment:
- [ **OPTIONAL** ] For best practices of modern UI/UX, I suggest you make your **logo** as a link so, once clicked it can lead the user to the home page.